### PR TITLE
create order issues

### PR DESCRIPTION
**Pull request by nishayaraj**
Pushed up to check and verify  issues with create order button

### CODE REVIEW

File: src/scripts/components/forms/createAnOrderForm.js [hunk 1]
```diff
@@ -4,83 +4,84 @@ import renderToDOM from '../../helpers/renderToDom';
...
+      <select
+        id="order-type"
+        class="form-select"
+        aria-label="Default select example">
+        <option
+          selected>
+          Select an Order Type
+        </option>
+        <option
+          value="1">
```
Comment (lesleyjkeyes) [inline src/scripts/components/forms/createAnOrderForm.js:68]: The value needs to read "Phone"

File: src/scripts/components/forms/createAnOrderForm.js [hunk 2]
```diff
@@ -4,83 +4,84 @@ import renderToDOM from '../../helpers/renderToDom';
...
+          Phone
+        </option>
+        <option
+          value="2">
```
Comment (lesleyjkeyes) [inline src/scripts/components/forms/createAnOrderForm.js:72]: The value needs to read "In-Person"

**Review by lesleyjkeyes — CHANGES_REQUESTED** [review]
see comments

**Review by gitNitroTitan — APPROVED** [review]
good job

**Review by lesleyjkeyes — APPROVED** [review]
looks good!

**Pull request merged**